File: assets/incdec.c
int main() {
    int* arr;
    int i;
    arr = malloc(10*SIZE);
    for (i = 0; i < 10; i++) {
        arr[i] = i;
    }
    i = 0;
    while (i < 10) {
        --arr[i++];
    }
    for (i = 0; i < 10; i++) {
        printf("%d\n", arr[i]);
    }
}

File: verify/calc/incdec.py
def expect(*args):
    arr = [*range(10)]
    i = 0
    while (i < 10):
        arr[i] -= 1
        i += 1
    res = ""
    for i in range(10):
        res += "{}\n".format(arr[i])
    return (0, res, "")

data = [
    [],
]
